projects/envoy/find_corpus.py

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/python
+import os, sys, re
+fuzzer_target = sys.argv[1].split('/')
+directory, fuzz_test = fuzzer_target[:-1], fuzzer_target[-1]
+directory_string = "/".join(directory)
+path = "../envoy/" + directory_string + "/BUILD"
+with open(path, "r") as f:
+        searchlines = f.readlines()
+        for i, line in enumerate(searchlines):
+                if fuzz_test in line: 
```

Comment:
Do you want to make this stricter? E.g. `if "name = \"fuzz_test\"" in line`?

Reply:
Well, `fuzz_test` is not string, it's a variable which has something like `h1_capture_fuzz_test`, I look for its occurrences as I traverse the file. Adding `name =` wouldn't be needed here. It can't be any more strict than this, this gets the exact occurrence.